src/server.conf.js

```diff
@@ -7,6 +7,7 @@ export const apiConf = {
         outlierDetection: '/outlierDetection',
         folloupAnalysis: '/dataAnalysis/followup',
         markDataValue: '/dataValues/followup',
+        updateMarkDataValue: '/dataAnalysis/followup/mark',
```

Comment:
We discussed this already a bit on Slack and the reason why currently the `Outlier Detection` section needs a different endpoint than the `Follow-up analysis` section is because the former uses UIDs but the latter uses integer IDs.

The long term fix would be to fix the `/dataAnalysis/followup` endpoint so it returns UIDs as well. A JIRA issue should be created for this and link included in the PR.

For the short term, the fix you have in place is good, but I would like the variable names to reflect their purpose slightly better. For example:
- `markDataValue` => `markOutlierDataValue`
- `updateMarkDataValue` => `markFollowUpDataValue`

Reply:
Jira issue regarding API returning integer IDs: https://jira.dhis2.org/browse/DHIS2-10658

Reply:
> A JIRA issue should be created for this and link included in the PR.

https://jira.dhis2.org/browse/DHIS2-10658